app/build.gradle

```diff
@@ -20,24 +20,24 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:27.1.1'
-    implementation 'com.android.support:design:27.1.1'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+    implementation 'androidx.appcompat:appcompat:1.0.0-beta01'
```

Comment:
Do we need to use beta versions? Are there stable versions that we can use?

Reply:
yeah I have made the changes. everything is updated to stable versions